sdk/src/loom2d/Display/Stage.ls

```diff
@@ -184,6 +207,31 @@ package loom2d.display
                     (modifiers | LoomKeyModifier.SHIFT) != 0 ));
         }
 
+        protected function onGameControllerButtonDownHandler(controller:int, button:int):void
+        {
+            broadcastEvent(new GameControllerEvent(GameControllerEvent.BUTTON_DOWN, controller, button));
```

Comment:
Ah, I have misunderstood this note. I have now pooled the axis events. Should all controller events be pooled? I believe button and connect/disconnect events are not problematic.